src/tests/test_utils.cpp

```diff
@@ -1232,6 +1232,8 @@ class UUID_Tests : public Test {
 
 BOTAN_REGISTER_TEST("utils", "uuid", UUID_Tests);
 
+#endif
```

Comment:
Drive-by repair: this is the closing part of `#if defind(BOTAN_HAS_UUID)` and it enclosed too much code.